qiita_ware/ebi.py

```diff
@@ -455,20 +376,20 @@ def generate_experiment_xml(self):
 
         Returns
         -------
-        xml.etree.Element
-            The root elelement of the generated ``ElementTree``
+        str
+            string with experiment XML values
         """
         study_alias = self._get_study_alias()
         experiment_set = ET.Element('EXPERIMENT_SET', {
```

Comment:
I know you have not modified this but, some of the values in the parameter dictionary are repeated in multiple places. Actually, it looks like the only thing that changes is the string after `/sra_1_3/` (i.e. `SRA.experiment.xsd` or `SRA.sample.xsd`, `SRA.run.xsd`, etc). Is it possible to create some globals and/or a function that returns this dictionary? It will reduce code duplication